src/shared/dissect-image.c

```diff
@@ -460,8 +460,12 @@ int dissect_image(
                 DissectedImage **ret) {
 
 #if HAVE_BLKID
-        sd_id128_t root_uuid = SD_ID128_NULL, root_verity_uuid = SD_ID128_NULL,
-                usr_uuid = SD_ID128_NULL, usr_verity_uuid = SD_ID128_NULL;
+#ifdef GPT_ROOT_NATIVE
```

Comment:
The variables are referenced slightly below, lines 511-518 - so there are more ifdefs needed

Reply:
indeed!